### PR TITLE
MAP 114 - Release the isd-creator2 github repo to public

### PR DESCRIPTION
**Pull request by DokurOmkar**
# PR Details
## Description

In this PR
- New public repo is created.
- All the code from isd-creator2 is moved into the new public repo.
- All the project files are updated to match the new repo.
- Docker and CI files are updated accordingly.

## Related Jira Key
[MAP-114](https://usdot-carma.atlassian.net/browse/MAP-114)

## Types of changes

- [ ] Defect fix (non-breaking change that fixes an issue)
- [ ] New feature (non-breaking change that adds functionality)
- [ ] Breaking change (fix or feature that cause existing functionality to change)

## Checklist:

- [ ] I have added any new packages to the sonar-scanner.properties file
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the [**CONTRIBUTING**](https://github.com/usdot-fhwa-stol/carma-platform/blob/develop/Contributing.md) document.
- [ ] I have added tests to cover my changes.
- [x] All new and existing tests passed.

[MAP-114]: https://usdot-carma.atlassian.net/browse/MAP-114?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: README.md
Comment (saina-ramyar) [inline README.md:1]: Let's update the build instructions

File: fedgov-cv-message-builder/LICENSE
Comment (saina-ramyar) [inline fedgov-cv-message-builder/LICENSE:1]: let's check with Kyle or someone if this license file is needed here. If so, we should update the dates and add similar file for other packages

File: README.md
Comment (DokurOmkar) [reply README.md:1]: Thank you for catching this. I have updated the build instructions accordingly.

File: fedgov-cv-message-builder/LICENSE
Comment (DokurOmkar) [reply fedgov-cv-message-builder/LICENSE:1]: I checked with Kyle and he told me that we can leave the LICENSE file as is without making any change to the dates.

**Review by saina-ramyar — APPROVED** [review]

**Review by saina-ramyar — APPROVED** [review]

**Pull request merged**